core/pom.xml

```diff
@@ -33,6 +33,7 @@
     <jcifs.version>1.3.3</jcifs.version>
     <woodstox-core-asl.version>4.4.1</woodstox-core-asl.version>
     <xmlunit.version>1.5</xmlunit.version>
+    <tomcat-version>10.1.24</tomcat-version>
```

Comment:
Can this be sync'd with the parent pom instead?

Reply:
@srallapa Also, can this property more specific to jdbc instead of <tomcat.version>?

Reply:
> Can this be sync'd with the parent pom instead?

Yes I have updated from the parent pom